base/Dockerfile.rhel8

```diff
@@ -53,9 +53,14 @@ RUN microdnf install yum \
     && yum clean all \
     && rm -rf /tmp/* \
     && rm -f requirements.txt \
-    && mkdir -p /etc/confluent/docker
+    && mkdir -p /etc/confluent/docker \
+    && useradd --no-log-init --create-home --shell /bin/bash appuser \
+    && chown appuser:appuser -R /etc/confluent/
 
-ADD target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/doc/* /usr/share/doc/${ARTIFACT_ID}/
-ADD target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/java/${ARTIFACT_ID}/* /usr/share/java/${ARTIFACT_ID}/
+ADD --chown=appuser:appuser target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/doc/* /usr/share/doc/${ARTIFACT_ID}/
```

Comment:
Is there any specific settings for user premission, or is it default?

Reply:
Since this just changes the ownership it doesn't change the permissions such such as read, write, or execute. the add and copy commands don't have an option to set the permissions, only to change ownership.